setup.cfg

```diff
@@ -96,7 +103,7 @@ sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,PACKAGES,LOCALFOLDER
 [mypy]
 python_version = 3.10
 strict_optional = True
-exclude=(.*_pb2|.*custom_types|packages/valory/connections/abci/tendermint/abci/|autonomy/data/Dockerfiles)
+exclude=(.*_pb2|.*custom_types|packages/valory/connections/abci/tendermint/abci/|autonomy/data/Dockerfiles|packages/valory/connections/http_client|packages/valory/connections/ledger|packages/valory/connections/p2p_libp2p_client|packages/valory/protocols/acn|packages/valory/protocols/contract_api|packages/valory/protocols/http|packages/valory/protocols/ledger_api)
```

Comment:
Same issue, but rather than package versions being different the config is different for mypy